$OpenBSD: patch-deps_v8_SConstruct,v 1.6 2011/12/07 12:18:01 sthen Exp $

- Remove extra zoom-zoom!
- Don't link with -lpthread but use -pthread instead.

--- deps/v8/SConstruct.orig	Wed Nov 16 13:43:22 2011
+++ deps/v8/SConstruct	Wed Nov 16 13:44:20 2011
@@ -94,7 +94,7 @@ LIBRARY_FLAGS = {
       'CPPDEFINES':   ['ENABLE_DISASSEMBLER', 'DEBUG'],
     },
     'mode:release': {
-      'CCFLAGS':      ['-O3', '-fomit-frame-pointer', '-fdata-sections',
+      'CCFLAGS':      ['-fomit-frame-pointer', '-fdata-sections',
                        '-ffunction-sections'],
     },
     'os:linux': {
@@ -358,7 +358,8 @@ MKSNAPSHOT_EXTRA_FLAGS = {
       'LINKFLAGS': ['-mt']
     },
     'os:openbsd': {
-      'LIBS': ['execinfo', 'pthread']
+      'LIBS': ['execinfo'],
+      'LINKFLAGS': ['-pthread'],
     },
     'os:win32': {
       'LIBS': ['winmm', 'ws2_32'],
